process/incident_management.md

```diff
@@ -0,0 +1,266 @@
+
+#### Responsibilities
+The investigators have the responsibility to describe the action they are taking in the dedicated IM channel to ensure that there is awareness on the steps being taken for the recovery of everyone involved in the process. This can also be done by someone else assisting in the investigation while the main investigator is busy with the recovery.
+
+### Incident Commander
+
+The key element for this role is to ensure that the Incident Commander is solely focused on "oiling the machinery" of incident recovery and will not participate in the hands-on resolution. In Fact the main goal for the Incident Commander is to deal with the overall coordination overhead of the incident recovery, with a cool head, while the investigators are focused 100% on performing the recovery. This will allow faster recovery, better communication flows and less errors during the stress of the incident management.
+
+The Incident Commander can elect someone to be the "scribe" during the incident. The duty of the scribe is to report in the RCA document all the important information (technical details, timeline, ...) while the Incident Commander is focused on helping with the coordination.
+
+#### Responsibilities
+The coordination work under the responsibility of the Incident Commander is as follows:
+1. If the incident has been uncovered by SRE and doesn't have an existing "external" ticket then the Incident Commander will create a ticket to track the overall life of the incident.
+2. Right after the Incident Commander will create a dedicated IM channel:
+  * Making sure that we have a dedicated IM channel opened specifically for the ongoing incident
+    1. Channel is public
+    2. Naming convention for the IM channel should start with the ticket number and the clusterid if available.
+       * Examples:
+         * #inc1096029-<CLUSTERID>
+         * #OHSS-1234-title
+    3. The description of the IM channel should record:
+      * Incident Commander
+      * Incident Tech Lead
+      * Bridge link
+      * A small description of the issue
+    4. Making sure the right people are invited in the IM channel (including customer support if need be)
+    5. Making sure that all communications and information flows are recorded in the IM channel (it will help with the Post Mortem Review process and making sure that we have the right timeline)
+  3. Using IM, post a message to the relevant channel with:
+    * **link to bridge**
+    * **link to the dedicated IM channel (see point 2.)**
+    * **short description of issue (which cluster, what’s failing)**
+  4. Post an incident notification to the customer(s)
+    * NOTE: If the ServiceLog facility is unavailable, work with the Customer Support Lead to open a proactive Support Case. If the incident may affect a large number of clusters / customers, the Customer Support Lead should escalate through their management chain to post a Customer Portal banner announcement.
```

Comment:
ServiceLog is very OSD–specific.

Reply:
use a generic term e.g. Customer Notifications System?